src/Cuba.jl

```diff
@@ -100,6 +109,15 @@ function generic_integrand!(ndim::Cint, x_::Ptr{Cdouble}, ncomp::Cint,
     func!(x, f)
     return Cint(0)
 end
+function generic_integrand_userdata!(ndim::Cint, x_::Ptr{Cdouble}, ncomp::Cint,
+                            f_::Ptr{Cdouble}, func!_and_userdata, nvec::Cint)
+    func!, userdata = func!_and_userdata
+    # Get arrays from "x_" and "f_" pointers.
+    x = unsafe_wrap(Array, x_, (ndim, nvec))
+    f = unsafe_wrap(Array, f_, (ncomp, nvec))
+    func!(x, f, userdata)
+    return Cint(0)
+end
```

Comment:
According to https://codecov.io/gh/giordano/Cuba.jl/compare/3ddf09bf2627745fab70fa90f0285350e5ac79d4...616a7a108c1c9bda05bbc0825aaf435c91a561af/diff this function is untested

Reply:
In the new commit, I have added some tests for vectorized integration with userdata. It should be sufficient to cover the untested function.